.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,132 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+  schedule:
+    - cron: "0 17 * * *"
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2.0.0"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@1.8.1"
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php-version }}"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v1.0.3"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}"
```

There is only one `composer.json`,  so this should do, hm?

```suggestion
          key: "php-${{ matrix.php-version }}-composer-${{ hashFiles('composer.json') }}"
```

---

This is what I have used since I started using GitHub Actions last year, and it is what is recommended, see https://github.com/actions/cache/blob/master/examples.md#php---composer.

---

I know, I used it this way as well for some time. But there's no point for having `**` here. There won't be multiple composer.jsons here.